src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/custom.py

```diff
@@ -822,3 +930,19 @@ def delete_certificate_issuer_admin(client, vault_base_url, issuer_name, email):
         vault_base_url, issuer_name, issuer.provider, issuer.credentials, org_details,
         issuer.attributes)
 # endregion
+
+# region storage_account
+
+
+def backup_storage_account(client, vault_base_url, storage_account_name, file_path):
+    backup = client.backup_storage_account(vault_base_url, storage_account_name).value
+    with open(file_path, 'wb') as output:
+        output.write(backup)
+
+
+def restore_storage_account(client, vault_base_url, file_path):
+    with open(file_path, 'rb') as file_in:
+        data = file_in.read()
+    return client.restore_storage_account(vault_base_url, data)
```

Comment:
The return should be inside the `with` statement I believe since that's where `data` is defined.